cpu/samd21/cpu.c

```diff
@@ -22,6 +22,10 @@
 #include "periph_conf.h"
 #include "periph/init.h"
 
+#ifndef VDD_MILLIVOLTS
+#define VDD_MILLIVOLTS 3300
+#endif
```

Comment:
So would you recommend/prefer *moving* the guarded `#define` from `cpu.c` to `periph_conf.h` or just *duplicating* it there?